docs/variables/countdown.md

```diff
@@ -31,11 +31,11 @@ This variable takes ***one*** *required* parameter that is a date string to coun
   * `02:00:00PM`
   * ...
 
-###### A list of more supported date and time formats can be found [**here**](https://github.com/araddon/dateparse)
+###### A list of more supported date and time formats can be found [**here**](https://github.com/araddon/dateparse).
```

Comment:
I don't think a heading tag is the correct way to write this markup. I get this styles the text correctly, but this impacts screen readers and doesn't aid accessibility imo.

Perhaps make a section with "Alternative formats" and put this sentence there, use a different way of formatting it, or something.

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/20#discussion_r788770539).